app/views/supported_permissions/index.html.erb

```diff
@@ -1,58 +1,72 @@
 <% content_for :title, "Permissions for #{@application.name}" %>
+<% content_for :breadcrumbs,
+   render("govuk_publishing_components/components/breadcrumbs", {
+     collapse_on_mobile: true,
+     breadcrumbs: [
+       {
+         title: "Home",
+         url: root_path,
+       },
+       {
+         title: "Applications",
+         url: doorkeeper_applications_path,
+       },
+       {
+         title: @application.name,
+         url: edit_doorkeeper_application_path(@application),
+       },
+       {
+         title: "Permissions",
+       }
+     ]
+   })
+%>
 
-<ol class="breadcrumb">
-  <li><%= link_to 'Applications', doorkeeper_applications_path %></li>
-  <li><%= link_to @application.name, edit_doorkeeper_application_path(@application) %></li>
-  <li class="active">Permissions</li>
-</ol>
-
-<div class="page-title">
-  <h1>Permissions for “<%= @application.name %>”</h1>
+<div class="govuk-form-group">
+  <%= render "govuk_publishing_components/components/button", {
+    text: "Add permission",
+    href: new_doorkeeper_application_supported_permission_path
+  } %>
 </div>
 
-<%= link_to new_doorkeeper_application_supported_permission_path, id: "add", class: "btn btn-success" do %>
-  <span class="glyphicon glyphicon-plus"></span> Add permission
-<% end %>
-
-<table class="table table-striped table-bordered add-top-margin">
-  <thead>
-    <tr class="table-header">
-      <th>
-        Permission name
-      </th>
-      <th>
-        Created
-      </th>
-      <th>
-        Can be delegated?
-      </th>
-      <th>
-        Is a default permission given to all new users?
-      </th>
-      <th>
-        Actions
-      </th>
-    </tr>
-  </thead>
-  <tbody>
-    <% @application.sorted_supported_permissions_grantable_from_ui.each do |supported_permission| %>
-      <tr>
-        <td class="name">
-          <%= link_to supported_permission.name, edit_doorkeeper_application_supported_permission_path(@application, supported_permission) %>
-        </td>
-        <td>
-          <%= supported_permission.created_at.to_date.to_fs(:govuk_date) %>
-        </td>
-        <td class="delegatable">
-          <%= supported_permission.delegatable? ? 'Yes' : 'No' %>
-        </td>
-        <td class="default">
-          <%= supported_permission.default? ? 'Yes' : 'No' %>
-        </td>
-        <td>
-          <%= link_to 'Edit', edit_doorkeeper_application_supported_permission_path(@application, supported_permission) %>
-        </td>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+<%= render "govuk_publishing_components/components/table", {
+  caption: "API users",
```

Comment:
I don't think this caption and label are correct / necessary.

Reply:
Addressed in 02bb452f32d3afb07850d59297146d4b4e3a73b5.